ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/forkchoice/ReadOnlyStore.java

```diff
@@ -33,6 +33,13 @@ public interface ReadOnlyStore {
 
   UInt64 getTime();
```

Comment:
I'd suggest renaming this (and the setter in MutableStore) to `getTimeSeconds` so that we're always being explicit about the units to avoid any confusion.